src/main/java/com/amadeus/airport/DirectDestinations.java

```diff
@@ -0,0 +1,62 @@
+package com.amadeus.airport;
+
+import com.amadeus.Amadeus;
+import com.amadeus.Params;
+import com.amadeus.Response;
+import com.amadeus.exceptions.ResponseException;
+import com.amadeus.resources.Destination;
+import com.amadeus.resources.Resource;
+
+/**
+ * <p>
+ *   A namespaced client for the
+ *   <code>/v1/airport/direct-destinations</code> endpoints.
+ * </p>
+ *
+ * <p>
+ *   Access via the Amadeus client object.
+ * </p>
+ *
+ * <pre>
+ * Amadeus amadeus = Amadeus.builder("clientId", "secret").build();
+ * amadeus.airport.directDestinations;</pre>
+ */
+public class DirectDestinations {
+  private final Amadeus client;
```

Comment:
Stupid question, why do we have a "private final" here instead of "private" like for other API?

Reply:
Thank you for noticing that! I have deleted the "final", which I think might be not necessary here. In JAVA, declaring instance variable as "final" is in order to make sure it will be definitely assigned at the end of any constructor of the class, and the value of this variable can't be modified once it is assigned. But just like other APIs, we will always assign the Amadeus instance at the constructor, thus the "final" could be not necessary.